internal/pkg/service/appsproxy/http/router.go

```diff
@@ -55,16 +54,15 @@ type Router struct {
 const (
 	providerCookie    = "_oauth2_provider"
 	selectionPagePath = "/_proxy/selection"
-
-	errorPagePath = "template/error.html.tmpl"
-	faviconPath   = "template/favicon.ico"
+	faviconPagePath   = "/_proxy/favicon.ico"
+	imagePagePath     = "/_proxy/rocket-flying.gif"
 )
 
-//go:embed template/*
-var templates embed.FS
+//go:embed template/* static/*
+var content embed.FS
```

Comment:
@hosekpeter  (cc @jachym-tousek-keboola) 

Here are some problems:

1. You are mixing together internal files `template/*` and public files `static/*`.
2. `static/` prefix of the files path make the implementation more complicated.
3. The main problem is: 

**You should mount all static assets in `mux` with one handler.**

**We don't want to name individual files there, it is unnecessary work.**

How exactly you achieve this is up to you.


----------------------

You should:

1. Create two variables.
```
//go:embed static
var static embed.FS

//go:embed template
var templates embed.FS
```

2. Mount all static files to the `mux`:
```go
import io/fs
...
staticDirFs := fs.Sub(static, "static")
staticHandler := http.FileServer(http.FS(staticDirFs))
mux.Handle("/_proxy/", http.StripPrefix("/_proxy/", staticHandler))
```

Read more about `fs.Sub`: https://www.iamyadav.com/blogs/a-guide-to-embedding-static-files-in-go

(maybe there are some typos in the code, I haven't tried it, I just wanted to point you in the right direction)